lib/ABILogs.js

```diff
@@ -38,10 +38,14 @@ addEvents();
 
 export async function getAllLogs(_logs) {
   const ethPrice = await getEthPrice();
-  return decodeLogs(_logs).map(log => {
-    const { coin, logo, decimals } = getToken(log.address);
+  return await decodeLogs(_logs).map(async log => {
+    let { coin, logo, decimals } = await getToken(log.address);
+    if(coin == "") {
+      let pair = await getPairAddress(log.address);
+      coin = pair.coin;
+    }
     let ethValue = 0;
-    let value;
+    let value = 1;
```

Comment:
It's important not to hardcode a value here if there's no value being transferred.